pkgs/games/minetest/default.nix

```diff
@@ -1,18 +1,18 @@
-{ stdenv, fetchgit, cmake, irrlicht3843, libpng, bzip2,
-  libjpeg, libXxf86vm, mesa, openal, libvorbis, x11 }:
+{ stdenv, fetchgit, cmake, irrlicht, libpng, bzip2, curl, libogg, jsoncpp,
```

Comment:
Little nitpick: I think we use no trailing commas
so

```
{ stdenv, fetchgit, cmake, irrlicht, libpng, bzip2, curl, libogg, jsoncpp
, libjpeg, libXxf86vm, mesa, openal, libvorbis, x11, sqlite }:
```
instead of
```
{ stdenv, fetchgit, cmake, irrlicht, libpng, bzip2, curl, libogg, jsoncpp,
  libjpeg, libXxf86vm, mesa, openal, libvorbis, x11, sqlite }:
```